website/components/footer/style.css

```diff
@@ -10,14 +10,12 @@
   }
 
   & a {
-    color: black;
-    opacity: 0.5;
-    transition: opacity 0.25s ease;
+    color: var(--gray-3);
     cursor: pointer;
     display: inline-block;
 
     &:hover {
-      opacity: 1;
```

Comment:
This contrast was an issue

Reply:
Should we keep the transition still?

Reply:
There is still a transition. It's transitioning to `black`, which is the same affect as before.

Reply:
Ahh - this is what I meant: https://github.com/hashicorp/vault/pull/11869/files/9a5c7f76b215c0fe3fe3db319f056d06b821c5cd#r654545408